recipes/pynumero_build/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "pynumero" %}
+{% set version = "0.0.0" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev : v0.0.0
+  git_url : https://github.com/santiagoropb/pynumero_build.git
+  
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [osx]
+  
+requirements:
+  build:
+
+    - cmake
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+about:
+  home: 
```

Comment:
The `home` entry needs to be filled in. You can probably just add the url for the git repo